src/modelldcatnotordf/modelldcatno.py

```diff
@@ -505,10 +507,14 @@ def _replaces_to_graph(self: InformationModel) -> None:
 
             for replaces in self._replaces:
 
-                _replaces = URIRef(replaces.identifier)
+                if isinstance(replaces, InformationModel):
+                    _replaces = URIRef(replaces.identifier)
+
+                    for _s, p, o in replaces._to_graph().triples((None, None, None)):
+                        self._g.add((_replaces, p, o))
 
-                for _s, p, o in replaces._to_graph().triples((None, None, None)):
-                    self._g.add((_replaces, p, o))
+                elif isinstance(replaces, str):
```

Comment:
Du sier at type er `Union[InformationModel, URI]`, men her sjekkes instance etter `InformationModel` og `str`, og testene bruker `InformationModel` og `str`. Hva skjer om du mater inn `URI`?

Reply:
Hmm.. Jeg skal sjekke. Men URI er jo en str (subklasse av str)

Reply:
Heisann igjen, da har jeg fått undersøkt dette. 

Dette er jo interessant. Det går fint å mate inn både strenger og URI'er:

```
informationmodel.replaces.append("https://example.com/informationmodels/3")
informationmodel.replaces.append(URI("https://example.com/informationmodels/3"))
```

Parseren takler begge disse to. :slightly_smiling_face:

Reply:
Jeg kan lage noen flere tester framover som viser slike eksempler bedre. :slightly_smiling_face: Er tilhenger av at testene viser litt realistisk/mulig/korrekt bruk av koden. :slightly_smiling_face: